diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileSize.kt

```diff
@@ -58,18 +51,9 @@ class FileSize(configRules: List<RulesConfig>) : DiktatRule(
          * Maximum allowed number of lines in a file
          */
         val maxSize = config["maxSize"]?.toLongOrNull() ?: MAX_SIZE
-
-        /**
-         * List of folders, files from which are ignored during the check. For example, for tests.
-         */
-        val ignoreFolders = config["ignoreFolders"]?.replace("\\s+".toRegex(), "")?.split(IGNORE_FOLDERS_SEPARATOR) ?: ignoreFolder
```

Comment:
Why now we don't have this parameter?

Reply:
We have decided that we no longer need it. It was originally introduced at the time when we didn't have exclusion mechanism, but now we can add excludes into plugin config or add file-level suppress for any rule, so special setting for file size is no longer required